pkg/reconcilermanager/controllers/rootsync_controller.go

```diff
@@ -426,6 +433,9 @@ func (r *RootSyncReconciler) Register(mgr controllerruntime.Manager, watchFleetM
 			builder.WithPredicates(predicate.ResourceVersionChangedPredicate{})).
 		// TODO: is it possible to watch with a label filter?
 		Watches(&source.Kind{Type: &rbacv1.RoleBinding{}},
+			handler.EnqueueRequestsFromMapFunc(r.mapObjectToRootSync),
+			builder.WithPredicates(predicate.ResourceVersionChangedPredicate{})).
+		Watches(&source.Kind{Type: &admissionv1.ValidatingWebhookConfiguration{}},
 			handler.EnqueueRequestsFromMapFunc(r.mapObjectToRootSync),
 			builder.WithPredicates(predicate.ResourceVersionChangedPredicate{}))
```

Comment:
You can't reuse the `mapObjectToRootSync` function here.
Basically, the function should map the `admission-webhook.configsync.gke.io` ValidatingWebhookConfiguration object to all RootSync objects.

Reply:
Done, added a new map function.